docs/src/user_guide/custom_model.md

```diff
@@ -1,45 +1,66 @@
 ## Definitions
 
-Within `Clapeyron`, it is possible to create your own model/equation of state whilst still using all of the property estimation tools we provide. In order to do this, you must create the model. We provide a macro to make it easy to set it up. If you wish to create a new model called CustomEoS, you just need call the `@newmodel` macro with three parameters:
+Within `Clapeyron`, it is possible to create your own model/equation of state whilst still using all of the property estimation tools we provide.
+In order to do this, you must create the model.
+We provide a macro to make it easy to set it up.
+If you wish to create a new model called CustomEoS, you just need call the `@newmodel` macro with three parameters:
 
 ```julia
 @newmodel CustomEoS CustomEosModel CustomEosParam
 ```
 
 1. Model struct name
-2. Model abstract type
-3. Model parameters struct
+1. Model abstract type
```

Comment:
seems like a typo?

Reply:
No, this is not a typo, see [here](https://www.markdownguide.org/basic-syntax/#ordered-lists). I prefer this, because then you don't "have to" change the numbering if you want/need to insert an item in the middle of the list.
I "enforce" this with

https://github.com/Mo-Gul/Clapeyron.jl/blob/4a4a964cfa663a2e1c3b96763ad305e381ce274b/docs/.markdownlint.json#L10

So, if you use [markdownlint](https://github.com/DavidAnson/markdownlint) in future you will be notified if you don't use this style 😄.

_(Remember, these are all *suggestions*. Thus, if you want to stick to the "old" style, please feel free to cherry-pick.)_